packages/app-store/typeform/components/InstallAppButton.tsx

```diff
@@ -0,0 +1,19 @@
+import type { InstallAppButtonProps } from "@calcom/app-store/types";
```

Comment:
Why do we need to add this file. InstallAppButton.tsx isn;t a required file and the install app button can work without it.
My aim is to remove all existing such files. Any new app created doesn't have this file.